statemachine/src/test/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DeleteHandlerTest.java

```diff
@@ -149,7 +149,9 @@ public void testThrowsStateMachineDoesNotExist_whenStateMachineDoesNotExist() {
         final ProgressEvent<ResourceModel, CallbackContext> response
                 = handler.handleRequest(proxy, request, null, logger);
 
-        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
+        assertThat(response.getStatus()).isEqualTo(OperationStatus.FAILED);
+        assertThat(response.getErrorCode()).isEqualTo(HandlerErrorCode.NotFound);
+        assertThat(response.getMessage()).contains(Constants.STATE_MACHINE_DOES_NOT_EXIST_ERROR_MESSAGE);
```

Comment:
nit: can we just construct the expected response and assert against it rather than attributes? 
are there properties that can't be captured in an expect?

Reply:
Agreed the tests can be cleaned up, but not in the scope of this PR. Just reverting the commit instead of refactoring anything related to the code.